internal/publish/r_package_descriptions.go

```diff
@@ -19,6 +21,12 @@ type lockfileErrDetails struct {
 	Lockfile string
 }
 
+const lockfileMissing = `Missing R lockfile %s. 
+The file must exist and be included in the deployment.
+Create the file listing your project dependencies.
+Or do an automatic scan with the help of the R Packages section
+of the Publisher extension and review the generated file`
```

Comment:
Whatever changes you end up making to the Python side based on my feedback I'd also make them here.